sled-agent/src/instance.rs

```diff
@@ -2003,7 +2004,7 @@ impl InstanceRunner {
             .with_log(self.log.clone())
             .with_underlay_vnic_allocator(&self.vnic_allocator)
             .with_zone_root_path(root)
-            .with_zone_image_paths(&["/opt/oxide".into()])
+            .with_file_source(&ramdisk_file_source("propolis-server"))
```

Comment:
Could `ramdisk_file_source` take a `ZoneType` or `ZoneKind` instead of a bare str?

Reply:
Ah no because propolis-server isn't an Omicron zone (what `ZoneType` and `ZoneKind` represent). Will add a comment here.